jdatetime/__init__.py

```diff
@@ -531,131 +562,67 @@ def __format__(self, format):
         """
         return self.strftime(format)
 
-    def strftime(self, format):
-        """format -> strftime() style string."""
-        # TODO: change stupid str.replace
-        # formats = {
-        #           '%a': lambda: self.j_weekdays_short[self.weekday()]
-        # }
-        # find all %[a-zA-Z] and call method if it in formats
-
-        # convert to unicode
-        try:
-            format = format.decode('utf-8')
-        except Exception:
-            pass
-
-        format = format.replace("%a", self.j_weekdays_short[self.weekday()])
+    def _strftime_replace_func(self, symbol, value, fallback=None):
```

Comment:
this function signature seems a bit cryptic, maybe we can add a docstring to help make it more clear?
also the `value` param, is actually a function that returns the value, so maybe we can use a name that reflects that? like `value_getter`/`val_func`/etc.?

Reply:
I've renamed `value` to `value_func` and also added a docstring to the method.